Packages/com.unity.inputsystem/Documentation~/ActionsEditor.md

```diff
@@ -44,11 +44,35 @@ If you’d like to delete all the default actions so that you can start from an
 * To delete an existing Action, either right-click it and select __Delete__ from the context menu.
 * To duplicate an existing Action, either right-click it and select __Duplicate__ from the context menu.
 
+## Action type and Control type
+
 If you select an Action, you can edit its properties in the right-hand pane of the window:
 
 ![Action Properties](Images/ActionProperties.png)
 
-### Editing Bindings
+#### Action Type
+
+The Action Type setting allows to to select between **Button**, **Value** or **PassThrough**.
+
+These options relate to whether this action should represent a discrete on/off button-style interaction or a value that can change over time while the control is being used.
+
+For device controls such as keyboard keys, mouse clicks, or gamepad buttons, select **Button**. For device controls such as mouse movement, a joystick or gamepad stick, or device orientation that provide continuously changing input over a period of time, select **Value**.
```

Comment:
Nitpick but "mouse click" is very different from "mouse button press". The former is an interaction and the second is a button press which this relates to. Hence recommend "mouse buttons" here or similar.

Reply:
Also a mouse click typically also relates not only to button but also to movement which may discard it as a click.